### PR TITLE
Upgrades rebrand cleanup

### PR DESCRIPTION
**Pull request by corwintines**
## Description
Implements feedback from Tim, Micah, and Dankrad from https://github.com/ethereum/ethereum-org-website/pull/4915#issuecomment-1018028898

> 'Eth1' is now the 'execution layer', which handles transactions and data. -> 'Eth1' is now the 'execution layer', which handles transactions and execution.
- In `/upgrades/`

> Shard chains spread the network's load across 64 new chains. -> Shard chains provide extra, cheaper, storage layers for applications and rollups to store data
- in `/upgrades/shard-chains/`

> They make it easier to run a node by keeping hardware requirements low. -> They enable layer 2 solutions to offer low transaction fees while leveraging the security of Ethereum
- in `/upgrades/shard-chains/`

> Remove this sentance 'The details of that are still being figured out.'
- in `/upgrades/shard-chains/

> Remove version 2 of shard chains information
- This is really the only part of sharding that is still on the roadmap. The second version has been put on hold for now. Perhaps it'd be worth getting a researcher to review this page and submit a PR to make sure it's accurate. `/upgrades/shard-chains/`

> Remove Cortex consensus client
- https://github.com/NethermindEth/cortex Cortex has been deprecated, so removing references to it from the codebase

- Add "Why can't we just use Eth2?" section to "What happened to Eth2?" section

### CODE REVIEW

File: src/content/upgrades/shard-chains/index.md [hunk 1]
```diff
@@ -51,36 +51,6 @@ Rollups are a "layer 2" technology that exists today. They allow dapps to bundle
 [More on rollups](/developers/docs/scaling/layer-2-rollups/)
 </InfoBanner>
 
-## Shard chains version 2: code execution {#code-execution}
```
Comment (minimalsm) [inline src/content/upgrades/shard-chains/index.md:54]: I'd prefer to get more input here before removing this section. ([see comment](https://discord.com/channels/595666850260713488/595701195843174434/932822345821859891) from @ralexstokes in R&D discord that implementing execution shard has not been abandoned)

File: src/pages/upgrades/get-involved/index.js [hunk 2]
```diff
@@ -355,41 +338,19 @@ const GetInvolvedPage = ({ data, location }) => {
...
-                background={client.background}
-                image={getImage(client.image(isDarkTheme))}
-                name={client.name}
-                description={client.description}
-                alt={translateMessageId(client.alt, intl)}
-                githubUrl={client.githubUrl}
-                hideStars={true}
-              />
-            ))}
+          {clients.map((client, idx) => (
```
Comment (minimalsm) [inline src/pages/upgrades/get-involved/index.js:341]: This includes Lodestar as a production-ready consensus client. Is that correct?

File: src/pages/upgrades/index.js [hunk 3]
```diff
@@ -382,6 +382,33 @@ const Eth2IndexPage = ({ data }) => {
...
+            </h3>
+            <p>
+              <Translation id="page-upgrades-why-cant-we-just-use-eth2-scam-prevention-description" />
+            </p>
+            <h3>
+              <Translation id="page-upgrades-why-cant-we-just-use-eth2-staking-clarity-title" />
+            </h3>
+            <p>
+              <Translation id="page-upgrades-why-cant-we-just-use-eth2-staking-clarity-description" />
+            </p>
```
Comment (minimalsm) [inline src/pages/upgrades/index.js:411]: I think the hierarchy here works better bringing these headings down a level.

Screenshot 2022-01-27 at 15 12 00

```suggestion
            <h3>
              <Translation id="page-upgrades-why-cant-we-just-use-eth2-title" />
            </h3>
            <h4>
              <Translation id="page-upgrades-why-cant-we-just-use-eth2-mental-models-title" />
            </h4>
            <p>
              <Translation id="page-upgrades-why-cant-we-just-use-eth2-mental-models-description" />
            </p>
            <h4>
              <Translation id="page-upgrades-why-cant-we-just-use-eth2-inclusivity-title" />
            </h4>
            <p>
              <Translation id="page-upgrades-why-cant-we-just-use-eth2-inclusivity-description" />
            </p>
            <h4>
              <Translation id="page-upgrades-why-cant-we-just-use-eth2-scam-prevention-title" />
            </h4>
            <p>
              <Translation id="page-upgrades-why-cant-we-just-use-eth2-scam-prevention-description" />
            </p>
            <h4>
              <Translation id="page-upgrades-why-cant-we-just-use-eth2-staking-clarity-title" />
            </h4>
            <p>
              <Translation id="page-upgrades-why-cant-we-just-use-eth2-staking-clarity-description" />
            </p>
```

**Review by minimalsm — CHANGES_REQUESTED** [review]
Looking good 👌

The main thing here is I'd really like to get a second opinion on removing execution sharding entirely. @ralexstokes seemed to have a different opinion on R&D discord very recently.

File: src/pages/upgrades/get-involved/index.js (hunk 2 above)
Comment (corwintines) [reply src/pages/upgrades/get-involved/index.js:341]: I believe its production ready after doing some looking around from Tim's comment yeah.

File: src/content/upgrades/shard-chains/index.md (hunk 1 above)
Comment (corwintines) [reply src/content/upgrades/shard-chains/index.md:54]: Will add this back in and can be apart of a separate issue if we want. Thanks for digging that up, good context to have.

**Comment by corwintines** [comment]
Updated based on your feedback @minimalsm, thanks!

File: src/pages/upgrades/get-involved/index.js (hunk 2 above)
Comment (samajammin) [reply src/pages/upgrades/get-involved/index.js:341]: Curious to hear from @carlbeek (& maybe others) on this. The PR on the Staking Launchpad hasn't yet been merged: https://github.com/ethereum/staking-launchpad/pull/436

Comment (wackerow) [reply src/pages/upgrades/get-involved/index.js:341]: Yeah also curious on the status of this. Otherwise the rest of this looks great!

Comment (wackerow) [reply src/pages/upgrades/get-involved/index.js:347]: Some Discord conversation leans towards identifying Lodestar as under review and audit. I have made this adjustment and sorted Lodestar to the end in the meantime, looking forward to removing this =)

**Review by wackerow — APPROVED** [review]

**Pull request merged**